doc/upmethodology-doc.tex

```diff
@@ -1130,18 +1130,18 @@ \section{Text formatting}
 
 \begin{itemize}
 \item \texttt{{\textbackslash}textsup\{text\}} \\
-	put a text as exponent in text mode instead of the basic \LaTeX exponent in math mode. In opposite to the standard \LaTeX\ macro \texttt{{\textbackslash}textsuperscript}, this macro adds an extra space after the macro when needed. \\
+	put a text as exponent in text mode instead of the basic \LaTeX\ exponent in math mode. In opposite to the standard \LaTeX\ macro \texttt{{\textbackslash}textsuperscript}, this macro adds an extra space after the macro when needed. \\
```

Comment:
The `\LaTeX`macro should be followed by a space macro. Otherwise no space will be rendered after the macro. For example, `\LaTeX abc` is rendered to `LaTeXabc` in the PDF file. Please revert your changes for all the occurrences of `\LaTeX`.